app/Kconfig

```
@@ -133,6 +133,14 @@ config SYSTEM_WORKQUEUE_STACK_SIZE
     default 4096 if SOC_RP2040
     default 2048
 
+config ZMK_LOW_PRIORITY_THREAD_STACK_SIZE
+    int "Low priority thread stack size"
+    default 768
+
+config ZMK_LOW_PRIORITY_THREAD_PRIORITY
+    int "BLE notify thread priority"
```

```suggestion
    int "Low priority thread priority"
```
A more generic description perhaps?